WordPress/WordPressShareExtension/ShareExtractor.swift

```diff
@@ -51,10 +51,10 @@ struct ShareExtractor {
 private extension ShareExtractor {
     var supportedExtractors: [ExtensionContentExtractor] {
         return [
+            ImageExtractor(),
             SharePostExtractor(),
             PropertyListExtractor(),
-            URLExtractor(),
-            ImageExtractor()
+            URLExtractor()
```

Comment:
I think I'd just flip these, but otherwise leave SharePost/PropertyList at the top, as they're more specific.